README.md

```diff
@@ -1,2 +1,38 @@
+---
+title: "CFSS hw01"
+author: "Yanning Cui"
+date: "January 9, 2017"
+output: pdf_document
+---
 # hw01
 Repository for first CFSS homework assignment
+
+##This is self introduction of Yanning!
+
+I am a **MAPSS** student and my concentration is *Sociology*.
+
+My research interest is computational content analysis using Twitter API.
+
+To be specific,
+
+- Twitter users' attitudes(**emotions** and **sentiments**) toward a specific event
+- Patterns of attitudes over an amount of time:
+   1.by region(the user's location)
+   2.by gender
+
```

Comment:
This seems great! I’m wondering how the outcome of your list “1. by region; 2. by gender” isn’t indented with your former list while in the code it is.